moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -232,6 +234,9 @@ class Servo
 
   // Map between joint subgroup names and corresponding joint name - move group indices map
   std::unordered_map<std::string, JointNameToMoveGroupIndexMap> joint_name_to_index_maps_;
+
+  // keep track of previously generated joint commands for constructing trajectory messages
+  std::deque<KinematicState> committed_commands_;
```

Comment:
```suggestion
  // rolling window of joint commands
  std::deque<KinematicState> joint_cmd_rolling_window_;
```

Reply:
Not a fan of the "committed" terminology. I think people normally think of this as a rolling window.

Reply:
I changed all references of "committed" to "joint_cmd_rolling_window"